feature/experimental/gribi/otg_tests/fib_failed_due_to_hw_res_exhaust_test/fib_failed_due_to_hw_res_exhaust_test.go

```diff
@@ -297,50 +326,15 @@ func TestFibFailDueToHwResExhaust(t *testing.T) {
 func sendTraffic(t *testing.T, args *testArgs) {
 	// Ensure that traffic can be forwarded between ATE port-1 and ATE port-2.
 	t.Helper()
-	t.Logf("TestBGP:start otg Traffic config")
-	flow1ipv4 := args.otgConfig.Flows().Add().SetName("Flow1")
-	flow1ipv4.Metrics().SetEnable(true)
-	flow1ipv4.TxRx().Device().
-		SetTxNames([]string{atePort1.Name + ".IPv4"}).
-		SetRxNames([]string{atePort2.Name + ".IPv4"})
-	flow1ipv4.Size().SetFixed(512)
-	flow1ipv4.Rate().SetPps(100)
-	flow1ipv4.Duration().Continuous()
-	e1 := flow1ipv4.Packet().Add().Ethernet()
-	e1.Src().SetValue(atePort1.MAC)
-	v4 := flow1ipv4.Packet().Add().Ipv4()
-	v4.Src().SetValue(atePort1.IPv4)
-	v4.Dst().Increment().SetStart(fibPassedDstRoute)
-	flow2ipv4 := args.otgConfig.Flows().Add().SetName("Flow2")
-	flow2ipv4.Metrics().SetEnable(true)
-	flow2ipv4.TxRx().Device().
-		SetTxNames([]string{atePort1.Name + ".IPv4"}).
-		SetRxNames([]string{atePort2.Name + ".IPv4"})
-	flow2ipv4.Size().SetFixed(512)
-	flow2ipv4.Rate().SetPps(100)
-	flow2ipv4.Duration().Continuous()
-	e2 := flow2ipv4.Packet().Add().Ethernet()
-	e2.Src().SetValue(atePort1.MAC)
-	v4Flow2 := flow2ipv4.Packet().Add().Ipv4()
-	v4Flow2.Src().SetValue(atePort1.IPv4)
-	v4Flow2.Dst().Increment().SetStart(fibFailedDstRoute)
 
-	args.otg.PushConfig(t, args.otgConfig)
-	time.Sleep(2 * time.Minute)
-	args.otg.StartProtocols(t)
-	otgutils.WaitForARP(t, args.ate.OTG(), args.otg.FetchConfig(t), "IPv4")
 	t.Logf("Starting traffic")
 	args.otg.StartTraffic(t)
 	time.Sleep(15 * time.Second)
 	t.Logf("Stop traffic")
 	args.otg.StopTraffic(t)
 
-	verifyTraffic(t, args, flow1ipv4.Name(), !wantLoss)
-	/*
-		if !deviations.GRIBISkipFIBFailedTrafficForwardingCheck(args.dut) {
-			verifyTraffic(t, args, flow2ipv4.Name(), wantLoss)
-		}
-	*/
+	verifyTraffic(t, args, fibPassedTraffic, !wantLoss)
+	verifyTraffic(t, args, fibFailedTraffic, wantLoss)
```

Comment:
Leave this deviation as it as unless there is confirmation from vendor who added this. 
'''
if !deviations.GRIBISkipFIBFailedTrafficForwardingCheck(args.dut) {
			verifyTraffic(t, args, flow2ipv4.Name(), wantLoss)
		}
'''